project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject nubank/matcher-combinators "1.3.1"
+(defproject nubank/matcher-combinators "1.3.2"
```

Comment:
`1.4.0` since it is a new behavior?

Reply:
Fine by me, but previous updates don't really follow that rule.